services/repology/repology-repositories.service.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { nonNegativeInteger } = require('../validators')
+const { BaseSvgScrapingService } = require('..')
+
+const schema = Joi.object({
+  message: nonNegativeInteger,
+}).required()
+
+module.exports = class RepologyRepositories extends BaseSvgScrapingService {
+  static get category() {
+    return 'platform-support'
+  }
+
+  static get route() {
+    return {
+      base: 'repology/repositories',
+      pattern: ':projectName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Repology - Repositories',
+        namedParams: { projectName: 'starship' },
+        staticPreview: this.render({ repositoryCount: '18' }),
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'in repositories',
```

Comment:
this is probably okay if you feel strongly about it, but i'd suggest just using `repositories` instead to be more consistent with other Shields badges (understand that repology use `in repositories` in their own badges). 

folks can always use the `?label=...` query param with Shields badges to change it to their liking, but just about all of our badges have single word messages/don't include prepositions